tokens/FIXX-074526/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "website": "https://fixxfi.online/",
+    "description": "Token utility of FixxFi. Rapairs made easy by installments",
+    "social": {
+        "email": "blockchain@fixxfi.com",
+        "blog": "https://fixxfi.medium.com/",
+        "twitter": "https://twitter.com/FixxToken",
+        "telegram": "https://t.me/fixxtoken",
+        "whitepaper": "https://fixxfi.medium.com/fixxfi-white-paper-25f67384ea09",
```

Comment:
Hello!
The comma on line 9 should be removed.
Please make sure to validate JSON with [https://jsonlint.com/](https://jsonlint.com/) before opening the PR.

Reply:
> Hello! The comma on line 9 should be removed. Please make sure to validate JSON with https://jsonlint.com/ before opening the PR.

Hello! 
Removed and validated the JSON. 
Thank you!